packages/config/src/projects/layer2s/ancient.ts

```diff
@@ -13,6 +14,7 @@ const upgradeability = {
 
 export const ancient: Layer2 = opStackL2({
   daProvider: CELESTIA_DA_PROVIDER,
+  badges: [Badge.VM.EVM, Badge.Stack.OPStack, Badge.DA.Celestia],
```

Comment:
could be added in the template

Reply:
Need dev help for this one, will be done separately